.gitignore

```diff
@@ -96,3 +96,9 @@ node-artifacts
 # AWS SAM generated
 test/lambda/.aws-sam
 test/lambda/env.json
+
+# files generated by tooling in drivers-evergreen-tools
+secrets-export.sh
+mo-expansion.sh
```

Comment:
😭🙌🚀

Reply:
ha, are you happy with this change or opposed to it?